src/Sylius/Bundle/ApiBundle/EventSubscriber/PriceableResourcesSubscriber.php

```diff
@@ -0,0 +1,81 @@
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ViewEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Sylius\Bundle\ApiBundle\Helpers\PriceTransformer;
+use function Clue\StreamFilter\fun;
+
+final class PriceableResourcesSubscriber implements EventSubscriberInterface
+{
+    public static function getSubscribedEvents()
+    {
+        return [
+            KernelEvents::VIEW => ['transformPrices', EventPriorities::POST_SERIALIZE],
+        ];
+    }
+
+    public function transformPrices(ViewEvent $event)
+    {
+        if (!$event->isMainRequest()) {
+            return;
+        }
+
+        $groups = $event->getRequest()->attributes->get('_api_normalization_context')['groups'];
+
+        if (in_array($groups, ['admin:order:read', 'shop:order:read'])) {
+            $function = 'changeOrderOutput';
+        } elseif (in_array($groups, ['admin:product_variant:read', 'shop:product_variant:read'])) {
+            $function = 'changeProductVariantOutput';
+        } else {
+            return;
+        }
+
+        $operation_type = $event->getRequest()->attributes->get('_api_normalization_context')['operation_type'];
+
+        $result = json_decode($event->getControllerResult(), true);
+
+        if ($operation_type === 'item') {
+            $event->setControllerResult(json_encode($this->{$function}($result)));
+        } elseif ($operation_type === 'collection') {
+            $transformedResult = [];
+            foreach ($result['hydra:member'] as $item) {
+                $transformedResult[] = $this->{$function}($item);
+            }
+            $result['hydra:member'] = $transformedResult;
+
+            $event->setControllerResult(json_encode($result));
+        }
+
+        return;
+    }
+
+    protected function changeOrderOutput($item)
+    {
+        $item['total'] = PriceTransformer::centimesToUnits($item['total']);
+        $item['itemsTotal'] = PriceTransformer::centimesToUnits($item['itemsTotal']);
+        $item['shippingTotal'] = PriceTransformer::centimesToUnits($item['shippingTotal']);
+        $item['orderPromotionTotal'] = PriceTransformer::centimesToUnits($item['orderPromotionTotal']);
+        foreach ($item['items'] as &$i) {
+            $i['unitPrice'] = PriceTransformer::centimesToUnits($i['unitPrice']);
+            $i['total'] = PriceTransformer::centimesToUnits($i['total']);
+            $i['subtotal'] = PriceTransformer::centimesToUnits($i['subtotal']);
+        }
+
+        return $item;
+    }
+
+    protected function changeProductVariantOutput($item)
+    {
+        $channel = $item['channelPricings']['FASHION_WEB'];
+        $item['channelPricings']['FASHION_WEB']['price'] = PriceTransformer::centimesToUnits($channel['price']);
+        $item['channelPricings']['FASHION_WEB']['originalPrice'] = PriceTransformer::centimesToUnits($channel['price']);
+
```

Comment:
As I told you, try to use constans instead of 'sometext' when you hardcode something. In this case in future we may change it by easiest way